src/assets/translations/en/main.json

```
@@ -970,7 +970,8 @@
   },
   "plugins": {
     "foxPage": {
-      "totalFoxValue": "Total FOX Value"
+      "totalFoxValue": "Total Fox Value",
```

```suggestion
      "totalFoxValue": "Total FOX Value",
```

no need to change in this PR, do it in future, but if the mockups say `Fox` i'd argue they're wrong and it should be `FOX`